src/ui/MessageTemplate/index.scss

```diff
@@ -0,0 +1,8 @@
+.sendbird-message-template__root * {
```

Comment:
@bang9 what is your opinion on this? I am not too familiar.

Reply:
resolved~~

Reply:
> @bang9 what is your opinion on this? I am not too familiar.
ㅋㅋㅋ me too

However, since the sub-elements of the message template can vary depending on the template, using a wildcard (*) selector seems like a good choice for now